client/components/Navbar.jsx

```diff
@@ -0,0 +1,60 @@
+// Temporary NavBar: to be changed
+// reference: https://ansonlowzf.com/how-to-build-a-material-ui-navbar/
+
+import * as React from 'react';
+import {
+  AppBar,
+  Toolbar,
+  IconButton,
+  List,
+  ListItem,
+  ListItemText,
+  Container,
+} from '@material-ui/core';
+import { Home } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  navbarDisplayFlex: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  navDisplayFlex: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  linkText: {
+    textDecoration: 'none',
+    textTransform: 'uppercase',
+    color: 'white',
+  },
+});
+
+const navLinks = [
+  { title: 'create', path: '/create' },
+  { title: 'browse', path: '/browse' },
+  { title: 'about', path: '/about' },
+];
+
+export const Navbar = () => {
+  const classes = useStyles();
+
+  return (
+    <AppBar position="static">
+      <Toolbar>
+        <Container maxWidth="lg" className={classes.navbarDisplayFlex}>
+          <IconButton edge="start" color="inherit" aria-label="home">
+            <Home fontSize="large" />
+          </IconButton>
+          <List component="nav" aria-labelledby="main navigation" className={classes.navDisplayFlex}>
```

Comment:
Awesome.  Will the `navLinks` shift to the hamburger menu when the screen size shrinks?

Reply:
I'm thinking that's the purpose of the `Toolbar` component?